frontend/public/components/_sysevent-stream.scss

```diff
@@ -101,8 +103,13 @@ $color-dark-border: #ddd;
 .co-sysevent__source {
   overflow: hidden;
   text-overflow: ellipsis;
+  white-space: nowrap;
+  flex: 1;
+}
+
+co-sysevent__count {
```

Comment:
missing a `.` at the start of this selector.  As a result, this rule isn't taking effect.  Is it needed?